src/Pages/Login/Login.tsx

```diff
@@ -1,10 +1,120 @@
-import React from 'react'
-import './Login.module.scss'
+import {
+  Box,
+  Button,
+  FormControl,
+  OutlinedInput,
+  Typography,
+} from "@mui/material";
+import axios from "axios";
+import { useForm } from "react-hook-form";
+import { useDispatch, useSelector } from "react-redux";
+import { Link, useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import {
+  selectAuthToken,
+  selectError,
+  setAuthToken,
+  setError,
+} from "../../Redux/Features/Auth/LoginSlice.ts";
+import baseUrl from "../../utils/Custom/Custom";
+import "./Login.module.scss";
 
 const Login = () => {
-  return <>
-    <div>Login</div>
-  </>
-}
+  const navigate = useNavigate();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+  const dispatch = useDispatch();
 
-export default Login
\ No newline at end of file
+  const onSubmit = (data: { email: string; password: string }) => {
+    axios
+      .post(`${baseUrl}/api/v0/admin/users/login`, data)
+      .then((res) => {
+        dispatch(setAuthToken(res.data.data.token));
+        toast.success("Welcome");
+        navigate("/dashboard");
+      })
+      .catch((err) => {
+        console.log(err);
+        dispatch(setError(err.message || "An error occurred"));
+        toast.error(err.message || "An error occurred");
+      });
+  };
+  return (
+    <>
+      <Box component="div">
+        {" "}
+        <Typography variant="h4" component="h4" sx={{ padding: "20px" }}>
+          <Box component="span" sx={{ color: "skyblue" }}>
+            Stay
+          </Box>
+          cation.
+        </Typography>
+      </Box>
+      <Box sx={{ padding: "30px 70px" }}>
+        <Box component="div">
+          <Typography variant="h4" component="h4">
+            Sign in
+          </Typography>
+          <Typography>
+            If you don’t have an account register <br /> You can
+            <Link to="/register"> Register here !</Link>
+          </Typography>
+        </Box>
+        <form onSubmit={handleSubmit(onSubmit)}>
+          <FormControl sx={{ width: "100%", margin: "20px 0" }}>
+            <OutlinedInput
+              type="email"
+              placeholder="Please type here ..."
```

Comment:
You need to change the placeholder text here to something like: 
Enter you E-mail